Automattic/wp-calypso
# e2e: Get e2e tests working with nav unification

#### Changes proposed in this Pull Request

* This PR changes the e2e tests to use the nav unification UI. It is not backward compatible with the old interface.

#### Testing instructions

* Run tests locally or in CI and they should use the nav unification and they should pass

Fixes https://github.com/Automattic/wp-calypso/issues/49250
